### ISSUE TITLE
Any change the TinyMce version will be updated to V6 ?

### ISSUE THREAD

**Issue by svendehens** [mohamedsabil83/filament-forms-tinyeditor#118]
### What happened?

Would you update this package to include TinyMce V6? It seems the package is using 5.10.7. at the moment. If not, is there a way I could bump this myself?

### How to reproduce the bug

...

### Package Version

2.2

### PHP Version

8.1

### Laravel Version

10.10

**Comment by faydanube** [comment]
> I did not find a time, but I plan to release a new major version, which will adopt v6 only or both v5 and v6, and the dev can choose between them from the configuration.

this one ?<https://github.com/amidesfahani/filament-tinyeditor>

**Comment by mohamedsabil83** [comment]
Yes, you can use it. It's a copy of my package with some modifications for v6. It is a good alternative until I publish the next major version.

**Comment by faydanube** [comment]
> Yes, you can use it. It's a copy of my package with some modifications for v6. It is a good alternative until I publish the next major version.

thx very much.

and

I've found a warning message and it can't be resolved via the obvious configuration file, so I'll try to fix it along with the next release.

Plugins:
- template, replaced by Advanced Template"
